src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -836,23 +987,81 @@ public CountryCodeProperties getCountryCodeISO3(final Geometry geometry, final b
      */
     public CountryCodeProperties getCountryCodeISO3(final Location location)
     {
-        return getCountryCodeISO3(JTS_POINT_CONVERTER.convert(location));
+        return this.getCountryCodeISO3(JTS_POINT_CONVERTER.convert(location));
+    }
+
+    /**
+     * @return {@link STRtree} grid index used by this {@link CountryBoundaryMap}
+     */
+    public STRtree getGridIndex()
+    {
+        return this.gridIndex;
```

Comment:
I agree on exposing it part, but that exposure is very limited and contained within small set of classes. Tree that being returned here could be big. Passing around copies could be safe, but expensive. 

Anyways classes, functionality around grid index and country boundary map need further cleanup. This can be addressed then I think.

Reply:
Anyone who has access to the CountryBoundaryMap can call this, which is any client who creates an Atlas. How about adding a comment in the javadoc that warns people against modifying the returned STRtree, as that can impact any slicing that happens.

Reply:
I'll limit it's scope and have a comment.